templates/README.example.md

```diff
@@ -36,7 +36,7 @@ bin/check
 
 This runs
 
-* keyword check (console.log, TODO, puts, ...)
+* keyword check (console.log, T0D0, puts, ...)
```

Comment:
I think we don't like the todo checks. Talk with roger about that. He has good arguments.